test/importer.js

```diff
@@ -30,13 +34,22 @@ describe("Importer", function() {
     
     it("Returns empty mappings on undefined", function() {
       var vpr = importer.flatSchema(undefined);
-      expect(vpr.schema).to.not.be.undefined();
-      expect(vpr.namesMapping).to.not.be.undefined();
+      expect(vpr.schema).to.deep.equal({});
+      expect(vpr.namesMapping).to.deep.equal({});
+    });
+
+    it("Parses fieldSchema", function() {
+      var vpr = importer.flatSchema(mapping.fieldSchema);
+      expect(vpr.schema).to.deep.equal({
+        'date': 'timestamp'
+      });
     });
     
     it("Parses tagsSchema", function() {
       var vpr = importer.flatSchema(mapping.tagsSchema);
-      expect(vpr.schema).to.not.be.undefined('No schema in result');
+      expect(vpr.schema).to.deep.equal({
+        'name': 'string'
+      });
     });
 
     // TODO: write test
```

Comment:
I think we also should add a test to config.js checking something about new tagsSchema